script.js

```diff
@@ -1,25 +1,37 @@
 var player = 1;
 var jogadas = 0;
 var ha_vencedor = false;
-var valida_posicao =  new Array(0,0,0,0,0,0,0,0,0);
-var marca_posicao = new Array();
+var valida_posicao = [
+    [false,false,false], // valida_posicao[0][0],valida_posicao[0][1],valida_posicao[0][2],
+    [false,false,false], // valida_posicao[1][0],valida_posicao[1][1],valida_posicao[1][2],
+    [false,false,false], // valida_posicao[2][0],valida_posicao[2][1],valida_posicao[2][2],
+];
 
-function clic(posicao_jogada){
+var marca_posicao =  [
+    [false,false,false],
+    [false,false,false], 
+    [false,false,false], 
+];
+
+document.getElementById("vez").innerHTML = 'Vez de O';
```

Comment:
_Strings_ exist in almost any programming language.

A string is often limited by quotes. Some languages require double quotes (`"`), some others, single quotes (`'`) and, some others - just like JavaScript - allow both.

However, in JavaScript, is a good practice to *always* use single quotes for strings. So, I would suggest to change all occurrences of double quotes to a single one.